node_modules/oae-core/editprofile/editprofile.html

```diff
@@ -19,10 +19,20 @@
                 <span aria-hidden="true">&#215;</span>
             </button>
         {/if}
-        <h3 id="editprofile-modal-title">__MSG__PROFILE_SUCCESSFULLY_UPDATED__</h3>
+        <h3 id="editprofile-modal-title">
+            {if isFromUpdate}
+                __MSG__PROFILE_SUCCESSFULLY_UPDATED__
+            {else}
+                __MSG__PLEASE_VERIFY_YOUR_EMAIL_ADDRESS__
```

Comment:
It might actually be better to always make this the title of this panel as it stresses what the actionable is for the user.